libs/utils/platforms/gem5juno.json

```diff
@@ -0,0 +1,30 @@
+{
+    "board" : {
```

Comment:
Other than board naming, is there any reason to have a gem5-specific platform file ? We should be able to use the same values as the "real" thing, right ? Same comment goes to the gem5-specific energy file.